tutorials/10_DebuggingREMIND.md

```diff
@@ -0,0 +1,183 @@
+  full.log and full.lst exist, so GAMS did run.
+! abort.gdx exists, containing the latest data at the point GAMS aborted execution.
+  fulldata.gdx exists, so at least one iteration was successful.
+  Number of iterations: 14
+  Modelstat: 4
+! full.log states: *** Status: Execution error(s)
+```
+
+If gams_runtime was short (say: up to 10 minutes), this implies an error right at the beginning of the run.
+If full.gms does not exist, it is an error in the preparation script, not in GAMS.
+
+If full.log exists, this is the next place to look at. Either open it in your favorite text editor or use the console, change directory to the output subfolder and type:
+```bash
+less full.log
+```
+and then type `G` to get to the end of the file. `q` finishes looking at the file.
+Another option is to use the editor `vi` by typing `vi full.log`, in which case `:q` closes the editor. `less` is generally faster when looking at large files.
+
+You may find:
+
+### Case 2a: compilation errors
+
+These are errors in the GAMS code. Have the next look at
+
+```
+less full.lst
+```
+Hit `G` to go to the end of the file and search backwards for errors whose lines start with four asterisks by typing:
+```
+/^\*\*\*\*
+```
+(You can also use `vi full.lst`, and if you add `map f /*\*\*\*<CR>` to your `/home/username/.vimrc` file, you can simply press `f`, which overwrites the search for single characters).
+Going upwards to the next matches using `N`, you should eventually find an error message (`n` searches downwards).
+The error is indicated directly at the point in the code where it occurs.
+It is worth looking at the `.gms` file this equation is part of, by searching for `SOF` (start of file) and `EOF` (end of file) marks above and below the equation.
+For fixing this error, the [section on compilation errors in the McCarl GAMS Guide](https://www.gams.com/mccarlGuide/fixing_compilation_errors_1.htm) may be helpful.
+
+You can either fix the error directly in the corresponding REMIND model file, or in `full.gms` in the output folder.
+In the first case, the PIK cluster provides the command `gamscompile` that can be run in the REMIND folder to see whether the change fixes the compilation error.
+Then, you can recreate the `full.gms` with running
+```
+Rscript start.R --reprepare
+```
+and selecting the run with the error. If you fix the code in the `full.gms` file (and only later transfer it to the REMIND directory), you can restart the run without the prepare part by running
+```
+Rscript start.R --restart
+```
+
+### Case 2b: execution errors
+
+For execution errors, again try to identify the location of the error in the `full.lst` file.
+If it points to a specific line number where the error occured, mark this number, then type `/` and paste it into the editor which is then searching for this number, and typing `n` (eventually multiple times) should bring you to the source code part at the beginning of the file.
```

Comment:
also here add that you are searching backward for this line number

Reply:
ok, changed `/` to `?` therefore